### PR TITLE
Add buttons to ContentDB in game bar and configure world

### PR DESCRIPTION
**Pull request by rubenwardy**
As requested by @sfan5, this PR adds a button to ContentDB in the game selection buttonbar. When clicked, the button opens ContentDB filtered to type "Game"

image

image

## To do

- [ ] Plus image that doesn't scale incorrectly

## How to test

Click it

### CODE REVIEW

**Comment by MoNTE48** [comment]
Oh my God. The button is so crooked and... weird...

**Comment by rubenwardy** [comment]
> Oh my God. The button is so crooked and... weird...

Yeah, irrlicht does weird stuff with image scaling

**Comment by MoNTE48** [comment]
> Yeah, irrlicht does weird stuff with image scaling

I know XD
1) in the image + is not centered!
2) maybe it should be bigger?
3) I hear him screaming - "put me on the right! I have to be on the right" 🙂

**Comment by rubenwardy** [comment]
this is center aligned in the source image at least, still has scaling artifacts:

image

Maybe the source image should be bigger than 32x32?

**Comment by sfan5** [comment]
MTG for example uses 128x128 textures in the creative mod to avoid scaling issues.

**Comment by rubenwardy** [comment]
With 128px image:

image

**Comment by rubenwardy** [comment]
here's 128px:

image

There's still scaling issues

**Comment by rubenwardy** [comment]
I've now added a button to the configure world menu: 
image

**Comment by rubenwardy** [comment]
Read the PR to change the creative flag, turns out they are slightly blurred to help things:

image

File: builtin/mainmenu/dlg_config_world.lua [hunk 1]
```diff
@@ -23,7 +23,50 @@ local function modname_valid(name)
 	return not name:find("[^a-z0-9_]")
 end
 
+local function init_data(data)
+	data.list = filterlist.create(
+			pkgmgr.preparemodlist,
```
Comment (sfan5) [inline builtin/mainmenu/dlg_config_world.lua:28]: too much indentation

File: builtin/mainmenu/dlg_config_world.lua [hunk 2]
```diff
@@ -85,11 +128,14 @@ local function get_formspec(data)
...
 	end
+
 	retval = retval ..
 		"button[3.25,7;2.5,0.5;btn_config_world_save;" ..
 		fgettext("Save") .. "]" ..
 		"button[5.75,7;2.5,0.5;btn_config_world_cancel;" ..
-		fgettext("Cancel") .. "]"
+		fgettext("Cancel") .. "]" ..
+		"button[9,7;2.5,0.5;btn_config_world_cdb;" ..
+			fgettext("Get Mods") .. "]"
```
Comment (sfan5) [inline builtin/mainmenu/dlg_config_world.lua:138]: "Get Mods" sounds kinda bad
maybe "Browse mods online" or "Find more mods"
or maybe no button at all?

Comment (KaylebJay) [reply builtin/mainmenu/dlg_config_world.lua:138]: Agreed, "Get Mods" sounds weird.

**Comment by rubenwardy** [comment]
image

**Pull request merged**